### ISSUE TITLE
can I download blast alignment results in bulk? I ...

### ISSUE THREAD

**Issue by wormbasehelp** [WormBase/website#5949]
*Help Desk query collected when no chat operators were online. Follow up required.*

can I download blast alignment results in bulk? I can from the ncbi site, but they just give old clone names instead of genome positions... or can I convert clone name and location to wormbase genome position in bulk somewhere?

**Reported by:** Greg (gste********************)
**Submitted from:** <a target="_blank" href="http://www.wormbase.org/http://www.wormbase.org/tools/support">http://www.wormbase.org/tools/support</a>
**Browser:** Chrome 61.0.3163.100

**Comment by chris-grove** [comment]
Hi Greg,


If you mean gene clone names like:


B0304.1
M05B5.5
W02C12.3


Yes, you can use WormMine to create a list of gene objects and then retrieve genomic coordinates for these. First, create a list of genes in WormMine like this:


https://www.dropbox.com/s/4d31kylm2cztsnz/Create_gene_list_in_WormMine.mov?dl=0


Then import the following XML query into WormMine (from the "Import query from XML" option on the WormMine QueryBuilder page, copy and paste the following query):


<query name="" model="genomic" view="Gene.primaryIdentifier Gene.secondaryIdentifier Gene.symbol Gene.chromosome.primaryIdentifier Gene.chromosomeLocation.strand Gene.chromosomeLocation.start Gene.chromosomeLocation.end" longDescription="" sortOrder="Gene.primaryIdentifier asc"> </query>


Then, in the QueryBuilder Model Browser, click on "Constrain" to the right of "WormBase Gene ID" 
and in the dialogue box that appears, click on the checkbox to the left of "Contained in list" 
and select the list you created from the dropdown menu. Click on "Add to query" and then click on "Show results".
You should then see a table of your genes and their chromosomal coordinates. You can export the table in 
a variety of formats.

Let us know if you have any questions.

Best,

Chris Grove
WormBase

**Comment by chris-grove** [comment]
From user:
"Thanks Christian!

That movie helps quite a bit, I used to use WormMart at lot but haven't
started using WormMine much.

I was trying to lookup all the Tc1 transposon insertions in the reference
genome to compare them. I just blasted a known sequence to see what came
out. It was a lot easier to download the resulting sequences from the NCBI
blast tool than wormbase, but then I wasn't sure if I was running off the
latest worm genome information and I couldn't easily get the chromosomal
locations for each sequence."
